includes/class-sensei-quiz.php

```diff
@@ -1497,8 +1496,9 @@ public static function start_quiz_questions_loop() {
 			}
 		}
 
-		// Fetch the questions.
 		$all_questions = Sensei()->lesson->lesson_quiz_questions( $quiz_id, 'publish' );
+
+		// Fetch the questions.
```

Comment:
I think this comment should be reverted to be before the `$all_questions` line.

Reply:
https://github.com/Automattic/sensei/pull/4606/commits/c0dccacdef11b3f6fbc123e5e350c888ca077291#diff-964a2368a54a4fd7dd7d8dfe593a8f74e690d028590dc23dbb8d1d4e5b68577fR1498